lib/object.js

```diff
@@ -60,6 +60,29 @@ export function set(target, path, value) {
   return target;
 }
 
+
+/**
+ * Gets a nested property of a given object.
+ *
+ * @param {Object} target The target of the get operation.
+ * @param {(string|number)[]} path The path to the nested value.
+ * @param {any} [defaultValue] The value to return if no value exists.
+ */
+export function get(target, path, defaultValue) {
+
+  var currentTarget = target;
+
+  forEach(path, function(key, idx) {
```

Comment:
`idx` is not used anywhere.

Reply:
https://github.com/bpmn-io/min-dash/pull/19/commits/796ccc6a1f33f0ebcf03e15bda75643c238d17df